client/components/forms/form-radios-bar/index.jsx

```diff
@@ -0,0 +1,45 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import FormRadioWithThumbnail from 'components/forms/form-radio-with-thumbnail';
+
+const FormRadiosBar = ( { isThumbnail, checked, onChange, items } ) => {
+	const addProps = ( Component, item, key ) =>
+		<Component key={ key } checked={ checked === item.value } onChange={ onChange } { ...item } />;
+
+	return (
+		<div className={ classnames( 'form-radios-bar', { 'is-thumbnail': isThumbnail } ) }>
+			{ items.map(
+				( item, i ) =>
+					isThumbnail
+						? addProps( FormRadioWithThumbnail, item, i )
+						: <FormLabel key={ i }>
```

Comment:
Also, what do you think to wrapping the elements that are part of the ternary in parens to improve the indentation?
Something like so:

```jsx
isSomething
  ? (
    <Something>
      <SomethingNested />
    </Something>
  )
  : (
    <SomethingElse>
      <SomethingElseNested />
    </SomethingElse>
  )
```
I'm not sure if it is actually much better so this is just an out-loud thought - keen to hear your thoughts though!